drf_integrations/fields.py

```diff
@@ -1,59 +1,13 @@
 import csv
-import django
 import io
-import logging
 from django import forms
-from django.conf import settings
 from django.core.exceptions import ValidationError
 from django.db import models
 from django.utils.encoding import force_str
-from django.utils.module_loading import import_string
 from django.utils.translation import ugettext_lazy as _
 
 from drf_integrations.utils import split_string
 
-logger = logging.getLogger(__name__)
-
-
-def get_json_model_field():
-    # if the Django version >= 3.2 then user the new default JSONField
-    if django.VERSION[0] >= 3 and django.VERSION[1] >= 2:
-        default = "django.db.models.JSONField"
-    else:
-        default = "django.contrib.postgres.fields.JSONField"
-
-    # Allow for override
-    model_field = getattr(settings, "DB_BACKEND_JSON_FIELD", default)
-    logger.info(f"Using django model JSONField: {model_field}")
-    try:
-        return import_string(model_field)
-    except ImportError:
-        raise ImportError(
-            "drf_integrations can only work with a backend that supports JSON fields, "
-            "please make sure you set the DB_BACKEND_JSON_FIELD setting to the "
-            "JSONField of your backend."
-        )
-
-
-def get_json_form_field():
-    # if the Django version >= 3.2 then user the new default JSONField
-    if django.VERSION[0] >= 3 and django.VERSION[1] >= 2:
-        default = "django.forms.JSONField"
-    else:
-        default = "django.contrib.postgres.forms.JSONField"
-
-    # Allow for override
-    form_field = getattr(settings, "DB_BACKEND_JSON_FORM_FIELD", default)
-    logger.info(f"Using django form JSONField: {form_field}")
-    try:
-        return import_string(form_field)
-    except ImportError:
-        raise ImportError(
-            "drf_integrations can only work with a backend that supports JSON fields, "
-            "please make sure you set the DB_BACKEND_JSON_FORM_FIELD setting to the "
-            "JSONField of your backend."
-        )
-
```

Comment:
Moved to the utils.py file because there were issues importing these functions from Platform